src/pages/workspaces/workspace/Data.js

```diff
@@ -223,18 +223,27 @@ const BucketContent = _.flow(
     activeStyle: { backgroundColor: colors.accent(0.2), cursor: 'copy' },
     onDropAccepted: uploadFiles
   }, [({ openUploader }) => h(Fragment, [
-    div([
-      _.map(({ label, target }) => {
-        return h(Fragment, { key: target }, [
-          makeBucketLink({ label, target, onClick: () => load(target) }),
-          ' / '
+    div({ style: { display: 'flex', justifyContent: 'space-between' } }, [
+      div([
+        _.map(({ label, target }) => {
+          return h(Fragment, { key: target }, [
+            makeBucketLink({ label, target, onClick: () => load(target) }),
+            ' / '
+          ])
+        }, [
+          { label: 'Files', target: '' },
+          ..._.map(n => {
+            return { label: prefixParts[n], target: _.map(s => `${s}/`, _.take(n + 1, prefixParts)).join('') }
+          }, _.range(0, prefixParts.length))
         ])
-      }, [
-        { label: 'Files', target: '' },
-        ..._.map(n => {
-          return { label: prefixParts[n], target: _.map(s => `${s}/`, _.take(n + 1, prefixParts)).join('') }
-        }, _.range(0, prefixParts.length))
+      ]),
+
+      div({ style: { justifyContent: 'flex-end' } }, [
```

Comment:
You should be able to remove `justifyContent`, since this doesn't have `display: 'flex'`.

Reply:
(In fact, you should be able to just remove this wrapper div altogether...)